packages/plugin-video/test/OdooVideo.test.ts

```diff
@@ -0,0 +1,28 @@
+import { OdooVideo } from '../src/OdooVideo';
+import { describePlugin, testEditor } from '../../utils/src/testUtils';
+import { BasicEditor } from '../../../bundles/BasicEditor';
+
+class TestVideoEditor extends BasicEditor {
+    constructor() {
+        super();
+        this.load(OdooVideo);
+    }
+}
+
+describePlugin(OdooVideo, () => {
+    describe('parser/parser', () => {
+        it('should parse and render an Odoo video', async () => {
+            await testEditor(TestVideoEditor, {
```

Comment:
Since you are using `describePlugin`, you should not need to create a new editor just to load a custom plugin. Replacing `TestVideoEditor` here by `BasicEditor` should work. If it doesn't then let me know because there would definitely be a bug somewhere.